python/cuml/test/test_dbscan.py

```diff
@@ -28,18 +28,22 @@
                  'no_structure']
 
 
+@pytest.mark.parametrize('max_bytes_per_batch', [10, 200, 2e6])
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
 @pytest.mark.parametrize('input_type', ['dataframe', 'ndarray'])
 @pytest.mark.parametrize('use_handle', [True, False])
-def test_dbscan_predict(datatype, input_type, use_handle):
+def test_dbscan_predict(datatype, input_type, use_handle, max_bytes_per_batch):
+
+    # max_bytes_per_batch sizes: 10=6 batches, 200=2 batches, 2e6=1 batch
```

Comment:
Probably would've been nice to have a c++ unit test to check these batch splits actually do the splits expected. (Not needed for current 0.7 cycle though.)

Reply:
I do agree that the c++ testing should specifically test the `computeBatch()` function in isolation. For now, in the spirit of time, I have at least lowered the batch size for the c++ test to force it to use 2 batches so that we can verify end to end correctness when batching. 

Here's the code that accomplishes this:

```
		// forces a batch size of 2
		size_t max_bytes_per_batch = 4 * sizeof(T) * 8;

		dbscanFit(handle, data, params.n_row, params.n_col, eps, min_pts, labels, max_bytes_per_batch, true);
```